core/server/data/migrations/versions/1.5/1-og-twitter-post.js

```diff
@@ -10,8 +10,8 @@ _private.handle = function handle(options) {
     let isAdding = type === 'Adding';
     let operation = isAdding ? commands.addColumn : commands.dropColumn;
 
-    return function (options) {
-        let connection = options.connection;
+    return function (opts) {
+        let connection = opts.connection;
 
         return connection.schema.hasTable(table)
             .then(function (exists) {
```

Comment:
Can you also change to `tablesExists` like in other files?

Reply:
Ah yeah :) consistency :muscle: